Warp.WebApp/Services/Entries/EntryService.cs

```diff
@@ -23,7 +23,7 @@ public EntryService(IDataStorage dataStorage, IImageService imageService, IRepor
     public async Task<Result<Guid, ProblemDetails>> Add(string content, TimeSpan expiresIn, List<Guid> imageIds, CancellationToken cancellationToken)
     {
         var now = DateTime.UtcNow;
-        var entry = new Entry(Guid.NewGuid(), content, now, now + expiresIn);
+        var entry = new Entry(Guid.NewGuid(), OpenGraphService.GetDescription(TextFormatter.Format(content)), now, now + expiresIn);
```

Comment:
Not an issue, but could you put the description to a separate variable first. There are a lot of things happens on a single line